python/simulators/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+pybind11_add_module(simulators simulators.cpp)
+
+set_target_properties( simulators PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/python/opm2 )
```

Comment:
Note: I that installed the Python module in `opm2` and not `opm` since there is a conflict with the `opm` module in `opm-common`. If I had called it `opm`, I could not have imported modules from both `opm` in `opm-common` and `opm` in `opm-simulators` in the same Python script